src/Nethermind/Nethermind.HealthChecks/NodeHealthService.cs

```diff
@@ -110,6 +141,37 @@ public CheckHealthResult CheckHealth()
                    _healthHintService.MaxSecondsIntervalForProducingBlocksHint();
         }
 
+        private bool _previousClCheckResult = true;
+        private DateTime _previousClCheckTime = DateTime.MinValue;
+        private int _lastMethodCallSuccesses;
+
+        private bool CheckClAlive(ICollection<(string Description, string LongDescription)> messages, string methodName)
+        {
+            var now = _api.Timestamper.UtcNow;
+            var forkChoiceUpdatedStats = _api.JsonRpcLocalStats!.GetMethodStats(methodName);
+
+            if (forkChoiceUpdatedStats.Successes == _lastMethodCallSuccesses)
```

Comment:
to say that CL is alive it doesn't have to be the success, but to say that our node is healthy it has to

Reply:
Why do we need that?